### ISSUE TITLE
Role based access broken in embed view

### ISSUE THREAD

**Issue by yoson7** [saltcorn/saltcorn#2682]
Trying to make a show view of info, all related to specific item. some info should be visible to all, some should be restricted to higher roles only.
Since in table, role based access refer only to write permissions but fileds are still visible, I created different view and configure it with minimum role of view, for admin. 
Than embed that view on the show view. 

Now it seems that since the minimum access to the show view is for regular users, it applies also on the restricted view, and everybody can see that info.

**Comment by yoson7** [comment]
Also, If I mark "click to edit" to a field in a view, anyone in who have access to the view can edit that field. even though that the field required higher role to write in that field.

**Comment by glutamate** [comment]
yes, i am aware of this. min role to run view is not checked in embedded views. Will include this is a summary of known issues and will fix for 1.2 (this will break a lot of applications)

**Comment by yoson7** [comment]
Thanks

**Comment by glutamate** [comment]
this is not strictly enforced
